spec/models/service_order_spec.rb

```diff
@@ -33,4 +33,38 @@
       end
     end
   end
+
+  describe ".short_his_name" do
```

Comment:
**#** for instance method
**.** for class method

so it will be ``describe "#short_hist_name"``

:hamster: